wd_capabilities.au3

```diff
@@ -174,11 +174,11 @@ Func _WD_CapabilitiesAdd($key, $value1 = Default, $value2 = Default)
 	If $value1 = Default Then $value1 = ''
 	If $value2 = Default Then $value2 = ''
 	Local Const $s_Parameters_Info = '     $key = ' & $key & '     $value1 = ' & $value1 & '     $value2 = ' & $value2
+	__WD_ConsoleWrite($sFuncName, @error, ': #' & @ScriptLineNumber & $s_Parameters_Info, $_WD_DEBUG_Full)
```

Comment:
Incorrect usage of __WD_ConsoleWrite

Reply:
Just saw your note re: `is for DEV testing`, but it still doesn't make sense to me unless you are using a custom version of __WD_ConsoleWrite.

Reply:
> Incorrect usage of __WD_ConsoleWrite

[fixing $_WD_DEBUG_Full usage](https://github.com/Danp2/au3WebDriver/pull/285/commits/c1740e2a3745bcd6028b0a4763f7b6c70e1e76c8)